tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py

```diff
@@ -197,3 +198,94 @@ def test004_account_add_delete_user(self):
                          [user['userGroupId'] for user in account['acl']])
 
         self.log('%s ENDED' % self._testID)
+
+    def test005_get_account_info(self):
+        """ ZRT-OVC-022
+        *Test case for getting account info*
+
+        **Test Scenario:**
+
+        #. Create an account (A1).
+        #. Get A1 and check its info.
+        """
+        self.log('%s STARTED' % self._testID)
+
+        self.log('Create an account (A1)')
+        openvcloud_ser_name = self.random_string()
+        ovc = self.robot.services.create(
+            template_uid="{}/openvcloud/{}".format(self.repo, self.version),
+            service_name=openvcloud_ser_name,
+            data={'name': self.random_string(),
+                  'location': self.location,
+                  'address': self.env,
+                  'token': self.iyo_jwt()}
+        )
+        ovc.schedule_action('install')
+
+        account_ser_name = self.random_string()
+        account_name = self.random_string()
+        account = self.robot.services.create(
+            template_uid="{}/account/{}".format(self.repo, self.version),
+            service_name=account_ser_name,
+            data={'name': account_name, 'openvcloud': openvcloud_ser_name}
+        )
+        account.schedule_action('install')
+
+        self.log('Get A1 and check its info')
+        acc_info = account.schedule_action('get_info').wait(die=True).result
+        self.assertEqual(account_name, acc_info['name'])
+        self.assertEqual(openvcloud_ser_name, acc_info['openvcloud'])
+        self.assertEqual('CXDRAU', acc_info['users'][0]['accesstype'])
+        ovc.schedule_action('uninstall')
+        account.schedule_action('uninstall')
+
+        self.log('%s ENDED' % self._testID)
+
+    def test006_set_vdcuser_groups(self):
+        """ ZRT-OVC-023
+        *Test case for setting vdcuser groups*
+
+        **Test Scenario:**
+
+        #. Create vdc user, should succeed.
+        #. Set user groups and check if it was set.
+        """
+        self.log('%s STARTED' % self._testID)
+
+        self.log(' vdc user, should succeed')
```

Comment:
why create new vdc user? why don't use the created user on setup ?

Reply:
Done.. because simply it is chance to verify dsl for creating user